sdk/test-utils/perf/src/options.ts

```diff
@@ -143,6 +145,15 @@ export const defaultPerfOptions: PerfOptionDictionary<DefaultPerfOptions> = {
       "Set to true to use the Node worker_thread API when running tests across multiple CPUs. Set to false to use child_process (default).",
     defaultValue: false,
   },
+  profile: {
+    description:
+      "Set to true to profile the perf test. When set to true, `cpus` will be overriden to 1.",
+    defaultValue: false,
+  },
+  "profile-filepath": {
```

Comment:
```suggestion
  "profile-output-path": {
```

NIT: I slightly prefer `--profile-output-path` to `--profile-filepath`.  Or even just `--profile-path`.

Reply:
going with profile-path